jobs/build/ocp/Jenkinsfile

```diff
@@ -711,52 +711,35 @@ ${failed_map}
                 } else if ( resp == "CONTINUE" ) {
                     echo "User chose to build fails are OK."
                     BUILD_EXCLUSIONS = failed_map.keySet().join(",") //will make email show PARTIAL
+                    BUILD_CONTINUED = true //simply setting flag to keep required work out of input flow
                     return true // Terminate waitUntil
                 } else { // ABORT
                     error( "User chose to abort pipeline because of image build failures" )
                 }
               }
             }
-        }
-
-        // Old method
-        if ( EARLY_LATEST_HACK.toBoolean() ) {
-            // Hack to keep from breaking openshift-ansible CI during US Eastern daylight builds. They need the latest puddle to exist
-            // before images are pushed to registry-ops in order for their current CI implementation to work.
-            OCP_PUDDLE = buildlib.build_puddle(
-                    PUDDLE_CONF,    // The puddle configuration file to use
-                    PUDDLE_SIGN_KEYS, // openshifthosted key
-                    "-b",   // do not fail if we are missing dependencies
-                    "-d",   // print debug information
-                    "-n"
-            )
-        }
-
-        stage( "push images" ) {
-            dir( "${env.WORKSPACE}/build-scripts/ose_images" ) {
-                TAG_LATEST = IS_SOURCE_IN_MASTER?"":"--nolatest"
-                if ( BUILD_VERSION != "3.8" && BUILD_VERSION != "3.9" && BUILD_VERSION != "3.7" ) { // Trying to get all of 3.7/3.8/3.9 building with oit
-                    sh "sudo ./ose_images.sh --user ocp-build push_images ${TAG_LATEST} --branch rhaos-${BUILD_VERSION}-rhel-7 --group base"
-                }
-                try {
-                    buildlib.print_tags("openshift3/ose")
-                } catch ( cex ) {}
+            
+            if ( BUILD_CONTINUED ) {
+              buildlib.oit """
+--working-dir ${OIT_WORKING} --group openshift-${BUILD_VERSION}
+${exclude} images:push --to-defaults
```

Comment:
why no --late-only?

Reply:
Wow... got to excited about removing early latest? Fixed it.